fixedpoint/fixedpoint.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+def decimal_to_fixed_p (num, width, int_bit, frac_bit):
+    '''
```

Comment:
Throw an error if the number is not representable using the number of fractionals bits you have. For example 10.8 cannot be represented using 2 fractional bits,

Reply:
Would raising exceptions also work?

Reply:
yeah that'd work

Reply:
Do I need additional checks to be able to merge?

Reply:
Don't know why they are not triggering. I'll merge.